gitsome/config.py

```diff
@@ -643,6 +652,11 @@ def save_config(self):
             parser.set(self.CONFIG_SECTION,
                        self.CONFIG_VERIFY_SSL,
                        self.verify_ssl)
+            # Update existing config files without enable_avatar correctly
+            if self.enable_avatar is not None:
```

Comment:
Can you move this up before `parser.set(self.CONFIG_SECTION,` next to the other blocks of code that set the config if the config is not None?

Reply:
Of course. Now that I look at it again, it fits a lot better above `if self.enterprise_url ...`. See 031e70989b47b1a6c789e659b85b73e9274ff71a